rllib/agents/a3c/tests/test_a3c.py

```diff
@@ -24,14 +24,15 @@ def test_a3c_compilation(self):
         num_iterations = 1
 
         # Test against all frameworks.
-        for fw in framework_iterator(config, ("tf", "torch")):
+        for fw in framework_iterator(config):
             config["sample_async"] = fw == "tf"
```

Comment:
Small suggestion: just do `fw in ["tf"]` since a3c will eventually be supported by tfe.